api/templates/gen/openapi.yaml

```diff
@@ -37,6 +37,58 @@ paths:
             $ref: '#/definitions/TemplatesIndexResponseBody'
       schemes:
       - https
+  /v1/documentation/openapi.json:
+    get:
+      tags:
+      - templates
+      summary: Download gen/openapi.json
```

Comment:
Ta summary nesedi. Malo by stacit pridat do `Files`, `function` ... `Description(...)`

Reply:
Alebo cez `Meta("swagger:summary", "...")` (neskusal som, co z toho ide).

Reply:
Aha. Description nepomohlo, summary tam zůstalo jak bylo. Ale ten `Meta("swagger:summary"` to vyřešil. 👍